units/provisioning_setup_host.go

```diff
@@ -388,26 +383,30 @@ func (j *setupHostJob) expandScript(s string, settings *evergreen.Settings) (str
 // Provision the host, and update the database accordingly.
 func (j *setupHostJob) provisionHost(ctx context.Context, h *host.Host, settings *evergreen.Settings) error {
 	grip.Infoln(message.Fields{
-		"runner":  HostInit,
+		"job":     j.ID(),
 		"host":    h.Id,
+		"distro":  h.Distro.Id,
 		"message": "setting up host",
 	})
 
+	incErr := h.IncProvisionAttempts()
```

Comment:
I don't think this changes anything. Previously we had been incrementing provision attempts in the error case.

Reply:
This is correct.

Reply:
It also looks like we're provisioning hosts too many times, e.g., https://evergreen.mongodb.com/host/sir-bygi66kg.